### ISSUE TITLE
Kuberhealthy Vulnerabilities found

### ISSUE THREAD

**Issue by naikajah** [Comcast/kuberhealthy#604]
Hello

We are using Kuberhealthy from sometime now. Recently on scanning kuberhealthy pods few vulnerabilities were detected by [NeuVector](https://neuvector.com/) 

kuberhealthy Image used: kuberhealthy/kuberhealthy:v2.2.0

Please find details of the vulnerabilities found

**15 High**

```
CVEs                       - Package                       - Package version with vulnerability
CVE-2019-16089  - linux/linux-libc-dev     - 4.19.98-1
CVE-2013-7445    - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19082  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19813  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-18814  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19072  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19816  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19814  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19815  - linux/linux-libc-dev     - 4.19.98-1
CVE-2019-19074  - linux/linux-libc-dev     - 4.19.98-1
CVE-2020-0041  - linux/linux-libc-dev      - 4.19.98-1

CVE-2020-8492  - python2.7                                      - 2.7.16-2+deb10u1
CVE-2020-8492  - python2.7/libpython2.7-minimal - 2.7.16-2+deb10u1
CVE-2020-8492  - python2.7/libpython2.7-stdlib    - 2.7.16-2+deb10u1
CVE-2020-8492  - python2.7/python2.7-minimal    - 2.7.16-2+deb10u1
```

Could you please let me know how we should go about resolving these?

Kind regards
Piyush

**Comment by jonnydawg** [comment]
Hi!
Could you please provide OS/kernel information?

**Comment by jonnydawg** [comment]
Hi @naikajah, there has been some changes made in light of these findings. Could you please re-scan the [patched image](https://hub.docker.com/layers/kuberhealthy/kuberhealthy/v2.3.1-alpha/images/sha256-9dc4cc1b571ee3dd0835b5a3dc6e7a065f20b6c7447852d67a20b5eb6e8e3a72?context=repo)?

**Comment by integrii** [comment]
A new patched image with an important fix is available at `kuberhealthy/kuberhealthy:v2.3.1-scratch`
